tao_compiler/mlir/disc/transforms/fusion_utils.h

```diff
@@ -137,58 +156,42 @@ int getNumResultOperands(Operation* op);
 // Here non-data users means DimOp, DeallocOp and ShapeOfOp.
 SmallVector<Operation*, 4> getValueUsers(Value v);
 
-// Represents a list of lmhlo ops that are going to be fused.
-class FusionPattern {
+struct TileInfo {
+  // Maps axis -> tile_size along this axis.
+  // select all the elements along the axis if tile_size ==
+  // ShapedType::kDynamicSize
+  DenseMap<int, int> tileSizes;
+
+  // Returns false if failed to merge.
+  bool merge(TileInfo& other);
+
+  // Returns false if failed to merge.
+  bool merge(int axis, int tileSize = ShapedType::kDynamicSize);
+
+  // Returns true if two TileInfo is equal.
+  bool isEqual(TileInfo& other);
```

Comment:
Not need this method actually. The default '==' method is enough for this struct.

Reply:
To fix.